jerry-core/parser/regexp/re-compiler.c

```diff
@@ -264,7 +264,7 @@ re_parse_alternative (re_compiler_ctx_t *re_ctx_p, /**< RegExp compiler context
       }
       case RE_TOK_CHAR:
       {
-        JERRY_TRACE_MSG ("Compile character token: %c, qmin: %d, qmax: %d\n",
+        JERRY_TRACE_MSG ("Compile character token: %" PRIu32 ", qmin: %" PRIu32 ", qmax: %" PRIu32 "\n",
```

Comment:
This will not produce the same output as the original: the first argument should not be rewritten from c to PRIu32 but the argument itself should be cast to char.